frontend/packages/shared/src/features/index.ts

```diff
@@ -1 +1 @@
-export { DataModelling } from './dataModelling';
+export { DataModelling } from '../../../../app-development/features/dataModelling';
```

Comment:
Do we actually need this?

Reply:
Automatic refactor gone wrong! Good catch, I'll remove it 🙂